src/web/app/components/question-submission-form/question-submission-form.component.ts

```diff
@@ -93,6 +94,7 @@ export class QuestionSubmissionFormComponent implements OnInit {
 
   visibilityStateMachine: VisibilityStateMachine;
   allowedToHaveParticipantComment: boolean = false;
+  isQuestionValid: boolean = true;
```

Comment:
Why we need to create a new field and trigger the change indirectly?

Reply:
I didn't think we could do the model assignment directly in the html template? Or is that possible? i.e. `(isValidEvent)="model.isValid=$event"`

Reply:
You should create a new method and fire the event with newly updated model.